docs/source/ProgrammingGuide/HierarchicalParallelism.md

```diff
@@ -64,6 +64,7 @@ Kokkos::TeamPolicy<SomeTag, ExecutionSpace>
 The team policy's `member_type` provides the necessary functionality to use teams within a parallel kernel. It allows access to thread identifiers such as the league rank and size, and the team rank and size. It also provides team-synchronous actions such as team barriers, reductions and scans.
 
 ```c++
+using Kokkos::Sum;
 using Kokkos::TeamPolicy;
 using Kokkos::parallel_for;
```

Comment:
Either we import all of them or none of them so adding `using Kokkos::Sum` looks fine.